src/foundry/client/data/documents/token.d.mts

```diff
@@ -48,27 +64,33 @@ declare global {
     get inCombat(): boolean;
 
     /**
-     * Define a sort order for this TokenDocument.
-     * This controls its rendering order in the PrimaryCanvasGroup relative to siblings at the same elevation.
-     * In the future this will be replaced with a persisted database field for permanent adjustment of token stacking.
-     * In case of ties, Tokens will be sorted above other types of objects.
+     * The Regions this Token is currently in.
+     * @privateRemarks TODO: Change Set<any> to Set<RegionDocuments> when RegionDocuments are implemented
      */
-    get sort(): number;
-
-    set sort(value);
+    regions: Set<any> | null;
 
+    /* -------------------------------------------- */
+    /*  Methods                                     */
+    /* -------------------------------------------- */
     protected override _initialize(options?: any): void;
     protected override _initialize(): void;
```

Comment:
I guess this isn't you but could you remove this second `_initialize`? There's no point to it because the `options?: any` makes it redundant.